LukeTHardy/backyard-buddy
# Lh finish styling

## Overview:

- Combined the Login and Register components into 1, styled them and animated login/register button

## Testing:

- Start the server and in your browser go to http://localhost:5173
- Verify login and register components are now one form with toggle tabs at the top
- Verify button animation works on hover, and click takes you to the homepage (if form is completed properly)
